### PR TITLE
Patch ActiveSupport in Rails 7.0.0+

### PR DESCRIPTION
**Pull request by artisr**
Based off 
https://github.com/globalize/globalize/pull/788
https://github.com/globalize/globalize/pull/790

cc: @ekampp @parndt

This is what failed for us: https://github.com/rails/rails/blob/v7.0.2/activesupport/lib/active_support/inflector/inflections.rb#L68

added ```I18n.respond_to?(:fallbacks)``` before ```I18n.fallbacks[locale].each do |k|```

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -6,14 +6,15 @@ RAILS_VERSIONS = %w[
...
+  7.0.0
 ]
 
 RAILS_VERSIONS.each do |version|
   appraise "rails_#{version}" do
     gem 'activemodel', version
     gem 'activerecord', version
     
-    if version =~ /^6/
+    if version =~ /^7/ || version =~ /^6/
```
Comment (ekampp) [inline Appraisals:17]: Doesn't this kind of matching quickly become cumbersome? wouldn't it be more advantageous to have the last `else` be the newest version?

**Comment by ekampp** [comment]
This is great. I would love to see this get merged in. Let me know, if there's something that I can help with.

File: Appraisals (hunk 1 above)
Comment (artisr) [reply Appraisals]: Done :)

**Comment by artisr** [comment]
:trollface:  Tests pass on my [machine](https://github.com/artisr/globalize/runs/5140561777?check_suite_focus=true) 

Let me know if this is acceptable or anything else needs changing.

**Comment by ekampp** [comment]
Nice. Thank you very much @mynameisdaniel, @parndt, and @artisr ❤️

**Comment by clemens** [comment]
Shouldn't globalize use the most recent 7.0.x release, i.e. at this point 7.0.2.2?

Also this change bothers me a bit: https://github.com/globalize/globalize/pull/792/files#diff-afb0c1fb4bfd16dfb55f678c90b38e39d5b65e412fd9d6cc08cfdcf4fe52507bR172-R173. It seems unrelated and also doesn't have a test.

**Comment by parndt** [comment]
> Any plans on when this will be released properly (= new gem release)?

version 6.1.0 released

**Pull request merged**